src/conduit/users.rs

```diff
@@ -1,26 +1,34 @@
+use super::errors::{FindError, InternalError};
 use crate::db::models::{NewUser, UpdateUser, User};
 use crate::db::schema::users;
-use crate::Repo;
-
+use bcrypt::BcryptResult;
+use diesel::pg::PgConnection;
 use diesel::prelude::*;
-use diesel::result::Error;
+use diesel::result::Error as DieselError;
 use uuid::Uuid;
 
-pub fn insert(repo: &Repo, user: NewUser) -> Result<User, Error> {
-    repo.run(move |conn| {
-        // TODO: store password not in plain text, later
+type Repo = crate::db::Repo<PgConnection>;
```

Comment:
It would be better to import the type alias, instead of redefining it.

Reply:
fixed with https://github.com/colinbankier/realworld-tide/pull/17/commits/d73c809d88a7b9525ce9c691508652fdd685a0b2